README.md

```diff
@@ -1,2 +1,108 @@
-# zenml-ui
-The repository that contains all source code for the ZenML UI.
+<!-- PROJECT LOGO -->
+<br />
+<div align="center">
+  <a href="https://zenml.io">
+    <img src="assets/zenml-dashboard-logo.gif" alt="Logo" width="400">
+  </a>
+
+  <h3 align="center">Open-source companion dashboard for
+   <a href="https://github.com/zenml-io/zenml">ZenML</a>.
+  </h3>
+  
+
+  <p align="center">
+    Manage and visualize your ML pipelines, stacks and artifacts in one place.
+    <br />
+    <a href="https://docs.zenml.io/"><strong>Explore the docs »</strong></a>
+    <br />
+    <div align="center">
+      Join our <a href="https://zenml.io/slack-invite" target="_blank">
+      <img width="25" src="https://cdn3.iconfinder.com/data/icons/logos-and-brands-adobe/512/306_Slack-512.png" alt="Slack"/>
+    <b>Slack Community</b> </a> and be part of the ZenML family.
+    </div>
+    <br />
+    <a href="https://zenml.io/features">Features</a>
+    ·
+    <a href="https://zenml.io/roadmap">Roadmap</a>
+    ·
+    <a href="https://github.com/zenml-io/zenml-dashboard/issues">Report Bug</a>
+    ·
+    <a href="https://zenml.io/discussion">Vote New Features</a>
+    ·
+    <a href="https://blog.zenml.io/">Read Blog</a>
+    ·
+    <a href="#-meet-the-team">Meet the Team</a>
+    <br />
+    🎉 Version 0.0.1 is out. Check out the release notes
+    <a href="https://github.com/zenml-io/zenml-dashboard/releases">here</a>.
+    <br />
+    <br />
+    <a href="https://www.linkedin.com/company/zenml/">
+    <img src="https://img.shields.io/badge/-LinkedIn-black.svg?style=for-the-badge&logo=linkedin&colorB=555" alt="Logo">
+    </a>
+    <a href="https://twitter.com/zenml_io">
+    <img src="https://img.shields.io/badge/-Twitter-black.svg?style=for-the-badge&logo=twitter&colorB=555" alt="Logo">
+    </a>
+  </p>
+</div>
+
+# ⚡ Getting Started
+
+## Installation
+
+`pip install zenml-dashboard`
+
```

Comment:
We should add a section here as to how its linked to the main ZenML repository

Basically, each ZenML python package comes with the build files (generated by `yarn build`) of a certain version of this dashboard (all build files of the dashboard can be seen in the releases section of this github). Therefore, each ZenML pypi package has bundled in it a corresponding version of this dashboard.

With the python package, you can then serve these static build files by doing:

```
zenml up
```

Which creates a local daemon that serves the files in a FastAPI server!

You can also deploy this dashboard on supported cloud providers with ZenML python package by using the command: 

```
zenml deploy
```